lib/stat_tracker.rb

```diff
@@ -1,4 +1,5 @@
 require 'csv'
+require 'pry'
```

Comment:
This might be a bit nit-picky but we should generally remove pry-related lines before merging

Reply:
good call ill make a note of it